kube-dash-image/kube-dash-image.kiwi.ini

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="6.5" name="_PRODUCT_-kube-dash">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>Kubernetes Dashboard running on a _DISTRO_ container guest</specification>
+  </description>
+  <preferences>
+    <type 
+      image="docker" 
+      derived_from="obsrepositories:/_BASEIMAGE_">
+      <containerconfig 
+        name="_PRODUCT_/kube-dash" 
+        tag="%%TAG%%" 
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/usr/bin/kubernetes-dashboard"/>
+      </containerconfig>
+    </type>
+    <version>1.0.1</version>
```

Comment:
All looks good to me, just this version is not aligned with the one in other images. Before, in pre-caasp v4 this was required to have the same major version as CaaSP, that's why all them are in the form of `4.0.x`. I believe this is not the case anymore and I think it only has effect on the resulting file name nowadays. However, just to be safe and in order to be coherent with other images, I'd say is better to set the version as `4.0.1`.

@jordi, this version was used by containment-rpm-docker to set the rpm package version, which is not used anymore in CaaSP, thus I believe we do not really care about that value, but I am not 100% sure it is not used in other places.

@sysrich, I think you are still using wrapped images in RPMs for Kubic, is the package version required to follow some versioning scheme?

Just as a not I noted heapster image is also having version 1.0.1, so far no one complained.

Reply:
Please use 4.0.* as version to be coherent.